test/integration/administration_spec.js

```diff
@@ -103,28 +103,40 @@ var mocker = function (database) {
         if (err) {
             console.log('Error initializing the database');
             process.exit(1);
+        } else {
+            return done();
         }
     });
 }
 
-async.each(test_config.databases, function (database, task_callback) {
+after(function (done) {
+    async.eachSeries(test_config.databases, function (database, task_callback) {
+        prepare_test.clearDatabase(database, databaseName, task_callback);
+    }, function (err) {
+        if (err) {
+            console.log(err);
+            return done();
+        } else {
+            return done();
+        }
```

Comment:
In the done is executed in both the if and the else you can extract it from the if